src/wp-admin/includes/class-wp-comments-list-table.php

```diff
@@ -231,9 +239,13 @@ public function no_items() {
 	}
 
 	/**
-	 * @global int $post_id
+	 * Returns an array of comment status links.
+	 *
+	 * @global int    $post_id
 	 * @global string $comment_status
 	 * @global string $comment_type
+	 *
+	 * @return string[] Array of comment status links.
```

Comment:
This can be more specific:


```suggestion
	 * @return array<string, string> Comment status HTML links keyed by view.
```